docs/hw/CAD-concept-schematics.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Build Armbian image
+parent: Operating System
+nav_order: 210
+---
+## Concept Schematics
+
+![BitBox Base 1](Base_render_angle_1.png "Concept Render of BitBox Base")
+
+![BitBox Base 2](Base_render_top_dimensions_1.png "Concept Render of BitBox Base - top view")
```

Comment:
broken link due to upper case `.PNG` in image filenames (this and next one below)